components/inputs/docs/input-select-styles.md

```diff
@@ -48,60 +48,159 @@ Toggling progressive disclosure is OK
 
 ## Applying styles to native select elements
 
-Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class.
+Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class. The styles account for several CSS pseudo-classes, including `disabled`, `focus` and `hover`.
 
-Note: in order for RTL to function correctly, make sure your component uses the `RtlMixin`.
+<!-- docs: demo code name:d2l-test-input-select -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+  class TestInputSelect extends LitElement {
+  static get styles() {
+    return [ selectStyles ];
+  }
+
+  render() {
+    return html`
+      <select
+        aria-label="Choose a dinosaur:"
+        class="d2l-input-select">
+        <option>Tyrannosaurus</option>
+        <option>Velociraptor</option>
+        <option>Deinonychus</option>
+      </select>
+    `;
+  }
+
+  }
+  customElements.define('d2l-test-input-select', TestInputSelect);
+</script>
+<d2l-test-input-select></d2l-test-input-select>
+```
+
+## Building a Custom Select Component
+
+In addition to the `d2l-input-select` styling and [native attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attributes), there are a few suggested Mixins and properties that we recommend using when creating a custom `Select` component.
+
+### Invalid
+
+Through the use of the [`aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid) attribute, you are able to mark the input as invalid, which shows up both in the styling along with being picked up by screenreaders.
+
+<!-- docs: demo name:d2l-test-input-select -->
```

Comment:
I think `name` is only needed if you want the properties table.
If we want these demos hooked up to the code sandbox, then they'll need unique `sandboxName` (Name of sandbox page if name is not set - unique identifier used for the url) and `sandboxTitle` (Human readable title for the sandbox page)

Reply:
Similar to my comment for RTL, I think we could be more direct here, something like this:

> Use the [`aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid) attribute to support screen reader users and apply our consistent error styles.

Should we also indicate how to wire up an error tooltip and ensure that's in the demo?

Reply:
Since they're just showing things off, just to show what it looks like, it probably doesn't need to be openable in sandbox, and if there should be one that's openable, then it would be the first one that would have several things in it.